### PR TITLE
Ios feat blokmap

### PR DESCRIPTION
**Pull request by feliciaan**
Confirm it works on android or ask if they care

### CODE REVIEW

File: iOS/Hydra/ActivitiesController.m [hunk 1]
```diff
@@ -537,4 +551,12 @@ - (void)dismissActionSheet:(id)sender
     self.datePicker = nil;
 }
 
+#pragma mark - RMPickerViewController Delegates
+- (void)pickerViewController:(RMPickerViewController *)vc didSelectRows:(NSArray  *)selectedRows {
```
Comment (FlashYoshi) [inline iOS/Hydra/ActivitiesController.m]: Is this still needed?

File: iOS/Hydra/ActivitiesController.m [hunk 2]
```diff
@@ -537,4 +551,12 @@ - (void)dismissActionSheet:(id)sender
...
+    //Do something
+}
+
+- (void)pickerViewControllerDidCancel:(RMPickerViewController *)vc {
```
Comment (FlashYoshi) [inline iOS/Hydra/ActivitiesController.m]: Is this still needed?

Comment (feliciaan) [inline iOS/Hydra/ActivitiesController.m]: Required method :(

**Comment by Silox** [comment]
> Confirm it works on android or ask if they care

@feliciaan Could you please elaborate?

**Comment by feliciaan** [comment]
@Silox that someone should build the project, so we know it works on android too. But if nobody has time then it shouldn't be merge blocking.

**Pull request merged**